example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -173,4 +178,66 @@ private void initListView() {
                 new int[]{R.id.last4, R.id.tokenId});
         listView.setAdapter(simpleAdapter);
     }
+
+    private void saveCreditCard() {
```

Comment:
Note: this method just got re-ordered for alphabetization sake. Still doing the same old Async method.